internal/provisioners/default/zz-default.provisioners.yaml

```
@@ -493,4 +493,128 @@
         type: ClusterIP
         ports:
         - port: 3306
-          targetPort: 3306
\ No newline at end of file
+          targetPort: 3306
+
+- uri: template://default-provisioners/mongo
+  type: mongo
```

This needs to be mongodb to line up with the same name used in score-compose and Humanitec.
```suggestion
  type: mongodb
```